docs/catalog/web/serve-images-in-modern-formats.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: dubrie
```

Comment:
Happy to take credit, but this should be your own github alias :-)

Reply:
To be honest I was unsure if this is the issuer (as I'm "only" converting it) or the convertor of the issue :)